src/hooks/api/component/useComponentList.ts

```diff
@@ -0,0 +1,27 @@
+import { InfiniteData, useInfiniteQuery } from "@tanstack/react-query";
+import { searchComponent } from "@/api/component";
+import { IPageData } from "@/types/component";
+
+interface IPageParam {
+  pageParam: number | unknown;
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export const useComponentList = () => {
```

Comment:
저는 보통 api관련 hook에는 이름을 tanstanck query 이름이랑 비슷하게 해줘요! 그러면 조금 덜 헷갈리더라구요 ㅎㅎ
```suggestion
export const useComponentListInfiniteQuery = () => {
```
좀 길어지긴 하지만.. 이런식으로요! 
이게 무조건 좋다는건 전혀 아니구.. 그냥 참고만 해주세용 ㅎㅎ

Reply:
직관적이고 좋은 네이밍인 것 같아서 [24ceec4](https://github.com/JECT-Study/Componote-FE/pull/70/commits/24ceec4989999e06adb3e3d6ad71586302c48d5f)에 반영했습니다!